bazel/MODULE.bazel

```diff
@@ -1,28 +1,65 @@
-module(name = "envoy_toolshed", version = "")
+module(
+    name = "envoy-toolshed",
```

Comment:
so when i was saying about kebab case it was more for the names (ie filepaths) of the examples

not sure about using it for bzlmod modules - i would say, probably lets dont, just on the basis that noone else seems to use that naming convention